DependencyInjection/CmfSeoExtension.php

```diff
@@ -250,20 +251,34 @@ private function loadSitemapHandling($config, XmlFileLoader $loader, ContainerBu
     {
         $loader->load('sitemap.xml');
 
-        $container->setParameter($this->getAlias().'.sitemap.configurations', $config['configurations']);
-
+        $configurations = $config['configurations'];
         foreach ($config['configurations'] as $key => $configuration) {
-            $container->setParameter($this->getAlias().'.sitemap.'.$key.'_configuration', $configuration);
+//            $container->setParameter($this->getAlias().'.sitemap.'.$key.'_configuration', $configuration);
+            if (isset($configuration['default_change_frequency'])) {
+                $definition = new DefinitionDecorator($this->getAlias().'.sitemap.guesser.default_change_frequency');
+                $definition->setArguments(array($config['configurations'][$key]['default_change_frequency']));
+                $definition->addTag('cmf_seo.sitemap.guesser', array(
+                    'sitemap' => $key,
+                    'priority' => 1,
+                ));
+                $container->setDefinition($this->getAlias().'.sitemap.guesser.'.$key.'.default_change_frequency', $definition);
+                unset($configurations[$key]['default_change_frequency']);
+            }
+            if (isset($config['defaults']['templates'])) {
+                foreach ($config['defaults']['templates'] as $format => $name) {
+                    if (!isset($configurations['templates'][$format])) {
+                        $configurations['templates'][$format] = $name;
+                    }
+                }
+            }
         }
```

Comment:
TODO: create configuration 'default' if it was not customized, so that the controller can handle the default sitemap without special case.